common/install_cuda_aarch64.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -ex
+
+function install_124 {
+  echo "Installing CUDA 12.4 and cuDNN 8.9 and NCCL 2.19.3 and cuSparseLt-0.5.2"
+  rm -rf /usr/local/cuda-12.4 /usr/local/cuda
+  # install CUDA 12.4.0 in the same container
+  wget -q https://developer.download.nvidia.com/compute/cuda/12.4.0/local_installers/cuda_12.4.0_550.54.14_linux_sbsa.run
+  chmod +x cuda_12.4.0_550.54.14_linux_sbsa.run
+  ./cuda_12.4.0_550.54.14_linux_sbsa.run --toolkit --silent
+  rm -f cuda_12.4.0_550.54.14_linux_sbsa.run
+  rm -f /usr/local/cuda && ln -s /usr/local/cuda-12.4 /usr/local/cuda
+
+  # cuDNN license: https://developer.nvidia.com/cudnn/license_agreement
+  mkdir tmp_cudnn && cd tmp_cudnn
+  wget -q https://developer.download.nvidia.com/compute/cudnn/redist/cudnn/linux-sbsa/cudnn-linux-sbsa-8.9.2.26_cuda12-archive.tar.xz -O cudnn-linux-x86_64-8.9.2.26_cuda12-archive.tar.xz
+  tar xf cudnn-linux-sbsa-8.9.2.26_cuda12-archive.tar.xz
+  cp -a cudnn-linux-sbsa-8.9.2.26_cuda12-archive/include/* /usr/local/cuda/include/
+  cp -a cudnn-linux-sbsa-8.9.2.26_cuda12-archive/lib/* /usr/local/cuda/lib64/
+  cd ..
+  rm -rf tmp_cudnn
+
+  # NCCL license: https://docs.nvidia.com/deeplearning/nccl/#licenses
+  # Follow build: https://github.com/NVIDIA/nccl/tree/v2.19.3-1?tab=readme-ov-file#build
+  git clone -b v2.19.3-1 --depth 1 https://github.com/NVIDIA/nccl.git
```

Comment:
This was recently updated to 2.20.5 https://github.com/pytorch/builder/pull/1724
```suggestion
  git clone -b v2.20.5-1 --depth 1 https://github.com/NVIDIA/nccl.git
```

Reply:
Going to keep in sync with https://github.com/pytorch/builder/pull/1720/files